### PR TITLE
Fixes #248 Removes fossasia_theme and integrates sphinx_fossasia_theme

### LINKED ISSUE #248: Remove fossasia_theme and make sphinx_fossasia_theme the new default theme
### This is a
- [ ] Bug report
- [ ] Feature request

### Current behaviour

The old deprecated fossasia_theme is still present in the repository. Meanwhile it has been replaced by sphinx_fossasia_theme and is available in PyPI.

### Expected behaviour

fossasia_theme should be removed.

### PR DESCRIPTION
**Pull request by pri22296**
## Description

- The older fossasia_theme has been removed.
- Earlier, on any error while theming, fosasia_theme was used as fallback. This was removed because having a theme from PyPI as the default would make yaydoc dependent on existence of that theme.
- The earlier implementation of loading local themes from _themes directory assumed that the directory would never be empty as it would always contain `fossasia_theme`. This is no longer the case so, made modifications according to that.

## Related Issue

Fixes #248 

## Motivation and Context

The older fossasia_theme was deprecated. The new theme is now being seperately developed at fossasia/sphinx_fossasia_theme and is available from PyPI

## How Has This Been Tested?

Demo deployment: https://yaydoc249.herokuapp.com/

## Types of changes

- [ ] Bug fix 
- [ ] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by pri22296** [comment]
@imujjwal96 @sch00lb0y Please review

File: views/index.jade [hunk 1]
```diff
@@ -40,6 +39,7 @@ block content
             option(value='pyramid') pyramid
             option(value='bizstyle') bizstyle
             option(disabled='') PyPi Custom Themes
+            option(value='sphinx_fossasia_theme') sphinx_fossasia_theme
```
Comment (imujjwal96) [inline views/index.jade]: how about we put this at top under heading `FOSSASIA's custom theme`? Will it make sense?

Comment (pri22296) [inline views/index.jade]: I suppose so. But that section will only contain a single theme right? would that look ok?

Comment (imujjwal96) [inline views/index.jade]: ok! leave it be then

**Comment by mariobehling** [comment]
Good job! Keep it up.

**Pull request merged**